rero/rero-ils
# SRU CQL search

* Adds CQL parser.

Co-Authored-by: Peter Weber <peter.weber@rero.ch>

## Why are you opening this PR?

- Implements a SRU CQL transformation to ES query string.
- This is an adaption of https://github.com/cheshire3/cheshire3

## How to test?

- poetry run console
> from rero_ils.modules.sru.cql_parser import parse
> query = parse('dc.anywhere all spam')
> query_string = query.to_es()

Verify the printout is a valid querystring!
Example: https://ilsdev.test.rero.ch/api/documents?q={$query_string}

## Code review check list
